code/modules/mob/living/living.dm

```diff
@@ -493,8 +493,25 @@
 		s_active.close(src)
 
 	if(lying && !buckled && prob(getBruteLoss()*200/maxHealth))
-
 		makeTrail(newloc, T, old_direction)
```

Comment:
you already lose blood by being dragged via this proc.

Reply:
I can't find it.

Reply:
makeTrail doesn't actually make you lose blood

Reply:
sure it does.

Reply:
Well i'll remove that!

Reply:
What's the point of that?